spinn_front_end_common/interface/interface_functions/local_tdma_builder.py

```diff
@@ -0,0 +1,283 @@
+# Copyright (c) 2020-2021 The University of Manchester
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+import math
+import logging
+
+from spinn_front_end_common.abstract_models.abstract_requires_tdma import \
+    AbstractRequiresTDMA
+from spinn_front_end_common.utilities import helpful_functions, \
+    globals_variables
+from spinn_front_end_common.utilities.exceptions import ConfigurationException
+from spinn_utilities.logger_utils import warn_once
+
+logger = logging.getLogger(__name__)
+
+
+class LocalTDMABuilder(object):
+    """ Builds a localised TDMA which allows a number of machine vertices
```

Comment:
Have you checked how Sphinx renders this comment?

Reply:
have i checked how anything is rendered in sphinx. how would i go about checking?